src/modsadmins/mods.ts

```diff
@@ -5,7 +5,6 @@ export const modsArray = [
   'not.me',
   'manuavalon',
   'maximovic96',
-  'd8nitemike',
```

Comment:
Uhh, not sure why this change is here. Can you pull master to make sure it's up to date in your branch?

Reply:
Fixed this as well.

Reply:
It's still showing differences here. You sure you pulled in master?

Reply:
Yep, this file matches the existing file in the code, I checked it.

Reply:
Ok all good